python/nammu/test/test_nammu.py

```diff
@@ -100,6 +103,10 @@ def selected_file_patch_arabic(a):
     return mockFile('resources/test/arabic.atf')
 
 
+def show_message_dialog(*args):
+    return 0
```

Comment:
Perhaps a brief docstring here? :) (just that this will be used to replace showing a dialog would be enough, I think)